packages/migrate/src/__tests__/MigrateDiff.test.ts

```diff
@@ -129,26 +72,7 @@ describe('migrate diff', () => {
       const result = await MigrateDiff.new().parse(['--from-empty', '--to-url', `file:${url}`, '--script'])
       expect(result).toMatchInlineSnapshot(``)
       expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
-      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
-        -- CreateTable
-        CREATE TABLE "Post" (
-            "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-            "title" TEXT NOT NULL,
-            "authorId" INTEGER NOT NULL,
-            FOREIGN KEY ("authorId") REFERENCES "User" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
-        );
-
-        -- CreateTable
-        CREATE TABLE "User" (
-            "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-            "email" TEXT NOT NULL,
-            "count1" INTEGER NOT NULL,
-            "name" TEXT
-        );
-
-        -- CreateIndex
-        CREATE UNIQUE INDEX "User_email_key" ON "User"("email" ASC);
-      `)
+      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
```

Comment:
Please replace `ctx.mocked['console.info']` with `ctx.mocked['process.stdout.write']`, so we don't lose track of the SQL script in the snapshot.